src/serializer/modules.js

```diff
@@ -169,18 +169,8 @@ export class ModuleTracer extends Tracer {
             const previousNumDelayedModules = this.statistics.delayedModules;
             do {
               try {
-                effects = realm.evaluateForEffects(() => {
-                  try {
-                    return performCall();
-                  } catch (e) {
-                    if (e instanceof Completion) return e;
-                    throw e;
-                  }
-                }, this);
-              } catch (err) {
-                if (err instanceof FatalError) effects = undefined;
-                else throw err;
-              }
+                effects = realm.evaluateForEffects(() => performCall(), this);
+              } catch (e) {}
```

Comment:
Might put a comment here about how we can't leak the exception so that we don't accidentally re-add it.

Reply:
Not quite the right place for that comment, but yeah, this code need comments. On the other hand it really needs to get deleted...